core-bundle/src/Resources/contao/forms/FormFileUpload.php

```diff
@@ -179,23 +181,23 @@ public function validate()
 
 		if ($arrImageSize = @getimagesize($file['tmp_name']))
 		{
-			$intImageWidth = Config::get('imageWidth');
+			$intImageWidth = $this->maxImageWidth ?: Config::get('imageWidth');
```

Comment:
```suggestion
			$intImageWidth = $this->maxImageWidth ? min($this->maxImageWidth, Config::get('imageWidth')) : Config::get('imageWidth');
```

I don't think as a user you should be able to increase the maximum of the system settings, only decrease. As a system admin, I want to be able to control maximum values.

Reply:
Basically I agree with you, but this is exactly the case I want to achieve with this PR. My customer wants to keep the limit (configuration settings) in all forms, only in this form it should be possible to upload bigger files (images). 🤔

Reply:
~~In that case you can remove/not add that field in the user rights.~~

Reply:
May be add a save callback, so that only admins can set it higher than the system limit? 🤔

Reply:
if that were useful the admin could simply hide the field for the back end user.

Reply:
> I don't think as a user you should be able to increase the maximum of the system settings, only decrease. As a system admin, I want to be able to control maximum values.

I agree with you. However, it is a special case when the file size is set higher for an upload field. If you need to set the system-wide maximum value higher for this special case, it also follows that all default upload fields would have to be limited manually.

Splitting imageWidth into a default value and an absolute maximum value could help. In this case a checkbox to activate the limitation in upload fields could be a solution to handle all cases.

Using a special value for the backend could additionally be more flexible.

Reply:
As discussed in the Contao call, we explicitly want to allow exceeding the maximum dimensions.